### PR TITLE
 Enabled autoescape for render context and using unescape for plain email parts.

### LINKED ISSUE #108: HTML escaping is used for context variables on plain text part and subject
The template engine is designed to escape fields that are handed over to the context dictionary. This is desirable for HTML, but not for plain text. Nevertheless also every context variable in ```{% block plain %} {% endblock %}``` or in ```{% block subject %} {% endblock %}``` is escaped.  E. g. ```“1 < 2 > 0 O'Leary "the master"”```  becomes ```“1 &lt; 2 &gt; 0 O&#39;Leary &quot;the master&quot;”```. 

Workaround is to use

```
{% block plain %} {% autoescape off %}
…
{% endblock %} {% endautoescape %}
```

But it would be better if autoescape is switched off by django-templated-email for blocks subject and plain in order to avoid unexpected results or at least write about autoescape off in the documentation.

### LINKED ISSUE #109: HTML is not autoescaped on html parts
Curiously #108 says HTML is escaped for plaintext and subject while I found the opposite problem: HTML is *not* escaped for html part which is a real danger and is not mentioned in the doc?

My solution would be to autoescape everything at first but afterwards unescape those plaintext parts.

I've now deployed such a fix but would like it to go upstream instead of forking, would you accept such a PR?

### PR DESCRIPTION
**Pull request by ComradeDOS**
This commits closes #108 and closes #109

### CODE REVIEW

**Comment by ComradeDOS** [comment]
Python3.3 not available for travis anymore.

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     py27-{flake8},
-    {py27,py33,py34,py35,py36}-django18,
+    {py27,py34,py35,py36}-django18,
```
Comment (fjsj) [inline tox.ini:4]: Drop those changes, keep only main `tox.ini`

**Review by fjsj — CHANGES_REQUESTED** [review]
thanks for the PR @ComradeDOS , @tuliolages is a maintainer that will make some fixes and merge this

**Comment by fjsj** [comment]
For now I'll close this one in preference of https://github.com/vintasoftware/django-templated-email/pull/120 which uses a simpler solution. Thanks for the PR and sorry we couldn't review it before.

**Pull request closed without merging**